<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmgQgisPluginOptionsBase</class>
 <widget class="QDialog" name="OmgQgisPluginOptionsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/om_icon_small.png</normaloff>:/om_icon_small.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Experiment directory</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>By default model data will be saved in your home directory in a sub-directory called '.omgui'. You can choose to override this location by specifying your own data directory below:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Data directory</string>
            </property>
            <property name="buddy">
             <cstring>leDataDirectory</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leDataDirectory"/>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolSelectDataDirectory">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/fileopen.png</normaloff>:/fileopen.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMapping">
      <attribute name="title">
       <string>Mapping</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Localities</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="lblLocalitiesDescription">
            <property name="text">
             <string>In the mapping tab, points will be displayed showing the presence and (if available) absence points used to create the model. You can choose how these points should be displayed on the map using the options below.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblLocalitySymbolSize">
            <property name="text">
             <string>Symbol size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblPresenceColour">
            <property name="text">
             <string>Colour for presence records</string>
            </property>
            <property name="buddy">
             <cstring>toolPresenceColour</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblPresenceColourPreview">
            <property name="styleSheet">
             <string>background-color: green;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="toolPresenceColour">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblAbsenceColour">
            <property name="text">
             <string>Colour for absence records</string>
            </property>
            <property name="buddy">
             <cstring>toolAbsenceColour</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lblAbsenceColourPreview">
            <property name="styleSheet">
             <string>background-color: red;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="toolAbsenceColour">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QDoubleSpinBox" name="spinLocalitySymbolSize">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>30.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>3.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Modeller Plugin</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Active plugin</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Modeller plugin</string>
            </property>
            <property name="buddy">
             <cstring>cboModellerPluginName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cboModellerPluginName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblOutputFormat">
            <property name="text">
             <string>Output format</string>
            </property>
            <property name="buddy">
             <cstring>cboOutputFormat</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cboOutputFormat"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="lblFormatNotes">
            <property name="text">
             <string>Format notes:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Plugin options</string>
         </property>
         <layout class="QGridLayout">
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="cbxModellerVerboseFlag">
            <property name="text">
             <string>Show verbose output if available (openModeller Debug Log level)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="cbxModellerLoggingFlag">
            <property name="text">
             <string>Store extra log data if possible (SOAP SEND, RECV and ERROR logs)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leWebServiceAddress">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblWebServiceAddress">
            <property name="text">
             <string>Web service address</string>
            </property>
            <property name="buddy">
             <cstring>leWebServiceAddress</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Additional info</string>
         </property>
         <layout class="QGridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="lblPluginSearchPath">
            <property name="text">
             <string>not set</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Plugin search path:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpProxy">
         <property name="title">
          <string>Use proxy for web access</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblProxyHost">
            <property name="text">
             <string>Host</string>
            </property>
            <property name="buddy">
             <cstring>cboModellerPluginName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leProxyHost"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblProxyPort">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>cboModellerPluginName</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leProxyPort"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblUser">
            <property name="text">
             <string>User</string>
            </property>
            <property name="buddy">
             <cstring>cboModellerPluginName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leProxyUser">
            <property name="toolTip">
             <string>Leave this blank if no proxy username / password are required</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblPassword">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>cboModellerPluginName</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leProxyPassword">
            <property name="toolTip">
             <string>Leave this blank if no proxy username / password are required</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>GBIF portal</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>GBIF portal url (e.g. data.gbif.org)</string>
            </property>
            <property name="buddy">
             <cstring>leGbifUrl</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="leGbifUrl"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="grpLogToFile">
         <property name="title">
          <string>Log to file</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>If you are experiencing problems and need to provide feedback to the openModeller Desktop project, you can enable logging to a file. The file can then be attached to any bug reports. This option is disabled by default.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Log file</string>
            </property>
            <property name="buddy">
             <cstring>leLogFile</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leLogFile"/>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolSelectLogFile">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/fileopen.png</normaloff>:/fileopen.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>530</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>leDataDirectory</tabstop>
  <tabstop>toolSelectDataDirectory</tabstop>
  <tabstop>spinLocalitySymbolSize</tabstop>
  <tabstop>toolPresenceColour</tabstop>
  <tabstop>toolAbsenceColour</tabstop>
  <tabstop>cboModellerPluginName</tabstop>
  <tabstop>cboOutputFormat</tabstop>
  <tabstop>leWebServiceAddress</tabstop>
  <tabstop>cbxModellerLoggingFlag</tabstop>
  <tabstop>cbxModellerVerboseFlag</tabstop>
  <tabstop>grpProxy</tabstop>
  <tabstop>leProxyHost</tabstop>
  <tabstop>leProxyPort</tabstop>
  <tabstop>leProxyUser</tabstop>
  <tabstop>leProxyPassword</tabstop>
  <tabstop>leGbifUrl</tabstop>
  <tabstop>grpLogToFile</tabstop>
  <tabstop>leLogFile</tabstop>
  <tabstop>toolSelectLogFile</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OmgQgisPluginOptionsBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OmgQgisPluginOptionsBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
